index.js

```diff
@@ -200,14 +226,14 @@ Resizer.prototype.changeSize = function (itemDetails) {
         .then(image => {
             return image
                 .scaleToFit(itemDetails.width, itemDetails.height) // resize
-                .quality(100) // set JPEG quality     
-                .getBufferAsync(Jimp.MIME_JPEG)
+                .quality(this.config.quality) // set PNG quality
+                .getBufferAsync(Jimp.MIME_PNG)
```

Comment:
Does it mean all images will be converted to PNG? Shouldn't it preserve the original format?

Reply:
I believe JIMP can get set to use the original format, I'll look into this